smithy-swift-codegen/src/test/kotlin/UnionGeneratorTests.kt

```diff
@@ -111,11 +112,13 @@ class UnionGeneratorTests {
         )
         val simpleUnionShape = simpleUnionShapeBuilder.build()
         val model = createModelFromShapes(simpleUnionShape)
-        val settings = model.defaultSettings()
-        val provider: SymbolProvider = SwiftCodegenPlugin.createSymbolProvider(model, settings)
+        val transformedModel = UnionIndirectivizer.transform(model)
```

Comment:
Had to run this transformer on the model so the test would pass.
Another example of how our test setup needs to be unified with our production logic.